main/webservices/api/v2.php

```diff
@@ -291,6 +291,38 @@
             $restApi->setMessageRead($messageId);
             $restResponse->setData(['status' => true]);
             break;
+        case Rest::CREATE_SESSION_FROM_MODEL:
+            if (
+                empty($_POST['modelSessionId']) || empty($_POST['sessionName']) || empty($_POST['startDate']) || empty($_POST['endDate'])
+            ) {
+                throw new Exception(get_lang('NoData'));
+            }
+            $modelSessionId = intval($_POST['modelSessionId']);
+            $sessionName = $_POST['sessionName'];
+            $startDate = $_POST['startDate'];
+            $endDate = $_POST['endDate'];
+            $extraFields = isset($_POST['extraFields']) ? $_POST['extraFields'] : [];
+            $newSessionId = $restApi->createSessionFromModel($modelSessionId, $sessionName, $startDate, $endDate, $extraFields);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line exceeds maximum limit of 120 characters; contains 128 characters](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4554450)

Reply:
the 120 characters limit is not a strict requirement, but function calls that are > 120 are generally split as one parameter per line.